hedyorg/hedy
# 🪲 Remove a source of client exceptions

A lot of client exceptions were originating from the embedded editor, trying to call `removeAttribute()` on an object that happened to be `null`.

The reason was an assertion that the result of `getElementById()` would never be `null`, where in fact, it was.

Replace with a null coalescing function call `?.removeAttribute()` so that the function call falls to the floor if the element doesn't exist.

**How to test**

Visit http://localhost:8080/embedded/1/?lang=nl&run=true&readOnly=false&fullWidth=false&program=IyBTY2hyaWpmIGpvdXcgY29kZSBoaWVy%22 and run any program. Observe that no exceptions get logged to the JavaScript console.

(cc @jpelay, I spied you [here](https://github.com/hedyorg/hedy/pull/5673) 😛 this is why `!` is dangerous, even you *think* this can never happen there's always a case you didn't think of -- better to program defensively or fail real hard with an explicit error message)
